source/index.ts

```diff
@@ -32,9 +35,9 @@ function validator(fullPath, relativePath) {
  * @param {validateCallback} next - the completion callback
  * @returns {void}
```

Comment:
param and return types should be trimmed in typescript, leaving only the name and description
for the return above, as it has no description, it should be trimmed